### PR TITLE
add keep_extend_vocab_only option

### PR DESCRIPTION
**Pull request by joey12300**
### PR types

New features
### PR changes

APIs
### Describe

* add keep_extend_vocab_only option
* simplify the embedding example doc

### CODE REVIEW

File: paddlenlp/embeddings/token_embedding.py [hunk 1]
```diff
@@ -58,14 +58,17 @@ class TokenEmbedding(nn.Embedding):
             The file path of extended vocabulary.
         trainable (object: `bool`, optional, default to True):
             Whether the weight of embedding can be trained.
+        keep_extend_vocab_only (object: `bool`, optional, default to True):
+            Whether keep the extend vocabulary only, only effective if provides extended_vocab_path
```
Comment (ZeyuChen) [inline paddlenlp/embeddings/token_embedding.py]: only effective if provides extended_vocab_path
->
will be effective only if `extendted_vocab_path` provided.

Comment (ZeyuChen) [reply paddlenlp/embeddings/token_embedding.py]: keep_extend_vocab_only  -> keep_extended_vocab_only   
extended才是形容词，不然是两个动词连用表意错误

**Review by ZeyuChen — CHANGES_REQUESTED** [review]
参数名修改

File: paddlenlp/embeddings/token_embedding.py (hunk 1 above)
Comment (joey12300) [reply paddlenlp/embeddings/token_embedding.py]: done

**Review by ZeyuChen — APPROVED** [review]
LGTM

**Pull request merged**